src/components/pages/settings/TeamsForm.js

```diff
@@ -23,13 +23,14 @@ class TeamsForm extends React.Component {
   }
 
   updateForm(team){
-    this.form = {
+    const updatedForm = {
```

Comment:
The previous version wasn't a mutation. You we're creating a new object. cc. @edbrett